### PR TITLE
RHDEVDOCS-2679 Document "Move Elasticsearch Operator from Operator ma…

### PR DESCRIPTION
**Pull request by rolfedh**
…turity level 3 to 4"

- Aligned team: Dev Tools
- For branches: 4.8+
- Jira: https://issues.redhat.com/browse/RHDEVDOCS-2679
- Direct link to doc preview: https://deploy-preview-36624--osdocs.netlify.app/openshift-enterprise/latest/logging/troubleshooting/cluster-logging-log-store-status?utm_source=github&utm_campaign=bot_dp#ref_cluster-logging-elasticsearch-cluster-status_cluster-logging-elasticsearch
- SME review: @igor-karpukhin 
- QE review: @kabirbhartiRH 
- Peer review: @robin-owen
- All reviews complete. Please merge now.
- FYI @libander @alanconway

### CODE REVIEW

File: logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.adoc [hunk 1]
```diff
@@ -7,6 +7,52 @@ include::modules/common-attributes.adoc[]
 toc::[]
 :toclevels: 2
 
+[id="status-of-elasticsearch-cluster"]
+== Status of Elasticsearch Cluster
+
+The Elasticsearch cluster uses the following fields to indicate its status.
```
Comment (rolfedh) [inline logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.adoc:13]: @igor-karpukhin Can we improve on this description of where and how the user can get the following messages?

Comment (igor-karpukhin) [reply logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.adoc:13]: @rolfedh I'd mention the dashboard in a Monitoring section in Openshift Console (`<cluster_url>/monitoring/dashboards/grafana-dashboard-cluster-logging`)

Comment (rolfedh) [reply logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.adoc]: Updated thanks. @igor-karpukhin  Please review and approve this PR.

**Review by igor-karpukhin — APPROVED** [review]
Looks good.

File: logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.adoc [hunk 2]
```diff
@@ -7,6 +7,53 @@ include::modules/common-attributes.adoc[]
...
+[source,terminal]
+----
+Total number of Namespaces.
+es_index_namespaces_total 5                                                                    Number of records against each namespace name.
+es_index_document_count{namespace="namespace_1"} 25
+es_index_document_count{namespace="namespace_2"} 10
+es_index_document_count{namespace="namespace_3"} 5
+----
+
+If Elasticsearch is missing an `admin-cert`, `admin-key`, `logging-es.crt`, or `logging-es.key`, it generates a status message similar to the following example:
```
Comment (robin-owen) [inline logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.adoc:50]: What are these? Files, fields, keys?....

**Review by kowen-rh — CHANGES_REQUESTED** [review]

File: logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.adoc [hunk 3]
```diff
@@ -7,6 +7,53 @@ include::modules/common-attributes.adoc[]
 toc::[]
 :toclevels: 2
 
+[id="status-of-elasticsearch-cluster"]
+== Status of Elasticsearch Cluster
+
+To get the status of the OpenShift Elasticsearch cluster, visit the Grafana dashboard in the *Monitoring* section of the {product-title} web console at
+`<cluster_url>/monitoring/dashboards/grafana-dashboard-cluster-logging`
+
+The `eo_elasticsearch_cr_cluster_management_state indicates` field indicates whether the Elasticsearch cluster is in a managed or unmanaged state. For example:
```
[the remark this answers is not in the document]
Comment (rolfedh) [reply logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.adoc:16]: Updated throughout. Thanks.

File: logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.adoc [hunk 4]
```diff
@@ -7,6 +7,53 @@ include::modules/common-attributes.adoc[]
...
+# eo_elasticsearch_cr_cluster_management_state - Shows if elasticsearch cluster that is in Managed state or Unmanaged state.
+eo_elasticsearch_cr_cluster_management_state{state="managed"} 1
+eo_elasticsearch_cr_cluster_management_state{state="unmanaged"} 0
+----
+
+The `eo_elasticsearch_cr_restart_total` field indicates the total number of times the Elasticsearch nodes have restarted for a certificate restart, rolling restart, or scheduled restart. For example:
+
+[source,terminal]
+----
+Total number of times the nodes restarted due to Cert Restart or Rolling Restart or Scheduled Restart.                                      eo_elasticsearch_cr_restart_total{reason="cert_restart"} 1
```
[the remark this answers is not in the document]
Comment (rolfedh) [reply logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.adoc:31]: This is the message the software generates. I'll give feedback to the developer that we should update the message.

[the remark this answers is not in the document]
Comment (rolfedh) [reply logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.adoc:31]: @igor-karpukhin Can we update this message in the software? "Total number of times the nodes restarted due to Cert Restart or Rolling Restart or Scheduled Restart. "

File: logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.adoc [hunk 5]
```diff
@@ -7,6 +7,53 @@ include::modules/common-attributes.adoc[]
...
+== Status of Elasticsearch Cluster
+
+To get the status of the OpenShift Elasticsearch cluster, visit the Grafana dashboard in the *Monitoring* section of the {product-title} web console at
+`<cluster_url>/monitoring/dashboards/grafana-dashboard-cluster-logging`
+
+The `eo_elasticsearch_cr_cluster_management_state indicates` field indicates whether the Elasticsearch cluster is in a managed or unmanaged state. For example:
+
+[source,terminal]
+----
+# eo_elasticsearch_cr_cluster_management_state - Shows if elasticsearch cluster that is in Managed state or Unmanaged state.
```
[the remark this answers is not in the document]
Comment (rolfedh) [reply logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.adoc:22]: I believe these messages are part of the software. I've put a question to the developer in Slack to confirm that this is the case and not a comment he inserted into the example output.

File: logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.adoc [hunk 6]
```diff
@@ -7,6 +7,55 @@ include::modules/common-attributes.adoc[]
 toc::[]
 :toclevels: 2
 
+[id="status-of-elasticsearch-cluster"]
+== Status of Elasticsearch Cluster
+
+To get the status of the OpenShift Elasticsearch cluster, visit the Grafana dashboard in the *Monitoring* section of the {product-title} web console at
+`<cluster_url>/monitoring/dashboards/grafana-dashboard-cluster-logging`.
```
Comment (kabirbhartiRH) [inline logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.adoc:14]: I am not able to see eo_elasticsearch_cr_cluster_management_state, eo_elasticsearch_cr_restart_total fields under monitoring/dashboards/grafana-dashboard-cluster-logging. Are we sure this is the correct path? @rolfedh

File: logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.adoc [hunk 7]
```diff
@@ -7,6 +7,53 @@ include::modules/common-attributes.adoc[]
...
+eo_elasticsearch_cr_restart_total{reason="rolling_restart"} 1
+eo_elasticsearch_cr_restart_total{reason="scheduled_restart"} 3
+----
+
+Elasticsearch has two index-level metrics:
+
+* The `es_index_namespaces_total` field indicates the total number of Elasticsearch index namespaces.
+* The `es_index_document_count` field indicates the number of records for each namespace.
+
+For example:
```
[the remark this answers is not in the document]
Comment (rolfedh) [reply logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.adoc:41]: Updated. Thanks.

File: logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.adoc [hunk 8]
```diff
@@ -7,6 +7,53 @@ include::modules/common-attributes.adoc[]
...
+[source,terminal]
+----
+Total number of times the nodes restarted due to Cert Restart or Rolling Restart or Scheduled Restart.                                      eo_elasticsearch_cr_restart_total{reason="cert_restart"} 1
+eo_elasticsearch_cr_restart_total{reason="rolling_restart"} 1
+eo_elasticsearch_cr_restart_total{reason="scheduled_restart"} 3
+----
+
+Elasticsearch has two index-level metrics:
+
+* The `es_index_namespaces_total` field indicates the total number of Elasticsearch index namespaces.
```
[the remark this answers is not in the document]
Comment (rolfedh) [reply logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.adoc]: Updated. Thanks.

File: logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.adoc [hunk 9]
```diff
@@ -7,6 +7,53 @@ include::modules/common-attributes.adoc[]
 toc::[]
 :toclevels: 2
 
+[id="status-of-elasticsearch-cluster"]
+== Status of Elasticsearch Cluster
+
+To get the status of the OpenShift Elasticsearch cluster, visit the Grafana dashboard in the *Monitoring* section of the {product-title} web console at
+`<cluster_url>/monitoring/dashboards/grafana-dashboard-cluster-logging`
```
[the remark this answers is not in the document]
Comment (rolfedh) [reply logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.adoc]: Updated. Thanks.

File: logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.adoc (hunk 2 above)
Comment (rolfedh) [reply logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.adoc:52]: Updated. Thanks.

File: logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.adoc (hunk 6 above)
Comment (rolfedh) [reply logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.adoc:14]: @igor-karpukhin what do you think?

File: logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.adoc [hunk 10]
```diff
@@ -7,6 +7,53 @@ include::modules/common-attributes.adoc[]
 toc::[]
 :toclevels: 2
 
+[id="status-of-elasticsearch-cluster"]
+== Status of Elasticsearch Cluster
```
[the remark this answers is not in the document]
Comment (rolfedh) [reply logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.adoc:11]: Good question. I'll discuss this with @igor-karpukhin.

File: logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.adoc (hunk 5 above)
[the remark this answers is not in the document]
Comment (rolfedh) [reply logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.adoc]: Thanks. The redundant text was not part of the command line output. I've removed it from the examples.

File: logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.adoc [hunk 11]
```diff
@@ -7,6 +7,53 @@ include::modules/common-attributes.adoc[]
...
+Elasticsearch has two index-level metrics:
+
+* The `es_index_namespaces_total` field indicates the total number of Elasticsearch index namespaces.
+* The `es_index_document_count` field indicates the number of records for each namespace.
+
+For example:
+
+[source,terminal]
+----
+Total number of Namespaces.
```
[the remark this answers is not in the document]
Comment (rolfedh) [reply logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.adoc]: Updates. Thanks.

File: logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.adoc (hunk 10 above)
[the remark this answers is not in the document]
Comment (rolfedh) [reply logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.adoc]: Updated. Thanks.

File: logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.adoc (hunk 6 above)
Comment (igor-karpukhin) [reply logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.adoc]: @sasagarw could you please post a link to the dashboard that you added with the https://issues.redhat.com/browse/LOG-1680?

**Review by kowen-rh — APPROVED** [review]
LGTM from a wording perspective. :+1:

**Pull request merged**